pyanaconda/modules/payloads/payload/rpm_ostree/installation.py

```diff
@@ -496,8 +498,10 @@ def name(self):
         return "Configure OSTree bootloader"
 
     def run(self):
-        self._move_grub_config()
-        self._set_kargs()
+        bootloader = STORAGE.get_proxy(BOOTLOADER)
+        if not bootloader.use_bootupd:
```

Comment:
Here you're calling the interface, so CamelCase `UseBootupd` once added as explained above.